display/social-sharing.php

```diff
@@ -11,36 +11,38 @@
 
 function nightingale_companion_sharing_caring( $content ) {
 	global $nightingale_companion_options;
-	$post_type = get_post_type();
-	$include   = array( 'post' );
-	$output    = '';
-	if ( is_single() && in_array( $post_type, $include ) ):
+	if ( ! empty( $nightingale_companion_options ) ) :
```

Comment:
While option check is ok there is no guarantee that array element exists... Shall we be more specific? ```if (isset($nightingale_companion_options['sharing_title']))``` ?

Reply:
Also using short if is not recommended in PHP functional code. Can we use standard curly brackets ? eg: ```if () { ... }```